src/building/how-to-build-and-run.md

```diff
@@ -242,3 +326,62 @@ everything up then you only need to run one command!
 
 `rm -rf build` works too, but then you have to rebuild LLVM, which can take
 a long time even on fast computers.
+
+## Build tools
+
+We've actually got quite a few tools that we use in the compiler's build system
+and for testing. To organize these, each tool is a project in `src/tools` with a
+corresponding `Cargo.toml`. All tools are compiled with Cargo (currently having
+independent `Cargo.lock` files) and do not currently explicitly depend on the
```

Comment:
I'm not sure which tools this "build tools" section is referring to. The only build tool I can think of is `x`. To me this seemed a little confusing, since it isn't clear the distinction between other directories in `src/tools`.

Reply:
It's also very old information which refers to https://github.com/rust-lang/rust/tree/f72bfe6661b35fd012fee100c673dafd1aec15f7/src/tools